gbfs.md

```diff
@@ -368,6 +378,85 @@ alerts            | Yes         | Array - alert objects each indicating a separa
 \- description     | Optional    | String - Detailed text description of the alert
 \- last_updated    | Optional    | Integer POSIX timestamp indicating the last time the info for the particular alert was updated
 
+### geofencing_zones.json
+This file is optional. All fields indicated as “Required” are only required if geofencing_zones.json is supplied.
+
+Field Name                           | Required    | Defines
+-----------------------------------  | ------------| ----------
+geofencing_zones{}                   | Yes         | A GeoJSON FeatureCollection (as described by the IETF [RFC 7946](https://tools.ietf.org/html/rfc7946#section-3.3)). Each geofenced zone and its associated rules and attributes is described as an object within the array of features, as follows.
+\- type                              | Yes         | String- “Feature” (as per IETF [RFC 7946](https://tools.ietf.org/html/rfc7946#section-3.3)).
+\- geometry                          | Yes         | A GeoJSON Multipolygon that describes where rides might not be able to start, end, go through, or have other limitations. A clockwise arrangement of points defines the area enclosed by the polygon, while a counterclockwise order defines the area outside the polygon [(right-hand rule)](https://tools.ietf.org/html/rfc7946#section-3.1.6). All geofencing zones contained in this list are public.
+\- properties{}                      | Yes		   | As defined below, describes travel allowances and limitations.
+&emsp;- name                         | Optional    | String- Name of the geofencing zone.
+&emsp;- rules[]                      | Optional    | Array that contains one object per rule as defined below. In the event of overlapping or colliding rule, the earlier defined rule (in order of the JSON file) takes precedence.
```

Comment:
One type of rule not currently represented in this PR (but mentioned in the discussion and used at Bird) is a speed-limited zone.  Perhaps we should add a `max_speed` field, with integer value.  We should decide what units we want to use (mi/hr, km/hr, something else).

Reply:
@wellorder I think the intention was to only cover rules related to parking in these zones and leave other types of rules/regulations as a responsibility of the MDS spec.

Reply:
@contra Whoa, I don't agree with this at all. GBFS should include everything that consumer-facing application can use and `max_speed` is certainly part of it : it impacts trip planning and is certainly interesting for the user to know in a UI. I though the speed was discarded because there were no implementer for it, but seems @wellorder is one of them now. 

I agree regulation/rules that aren't user facing at all can be left to MDS (like vehicle caps).